src/main/kotlin/pt/up/fe/ni/website/backend/controller/AuthController.kt

```diff
@@ -14,6 +16,9 @@ import pt.up.fe.ni.website.backend.service.AuthService
 @RestController
 @RequestMapping("/auth")
 class AuthController(val authService: AuthService) {
+    @field:Value("\${page.recover-password}")
+    private lateinit var recoverPasswordPage: String
```

Comment:
Shouldn't this be passed by config and config properties or is it bloat to use? @BrunoRosendo @bdmendes

Reply:
What do you mean? It is passed by the config property